### PR TITLE
Fixed recoverSigner()

### PR DESCRIPTION
**Pull request by anthonygd**
## Scope

Since Proposals are signed with a prefix we need to prefix the pclaims message with the same before hashing for ecrecover()

## Why?

Each object type's signature has a distinct prefix that needs to be specified before we can recover the signer address.

### CODE REVIEW

File: src/facets/AccusationLibrary.t.sol [hunk 1]
```diff
@@ -3,74 +3,31 @@ pragma solidity >= 0.5.15;
...
-
-    function testAccuseMultipleProposal() public {
+    function testSignNoPrefix() public {
         AccusationMultipleProposalFacet f = new AccusationMultipleProposalFacet();
 
-        // bytes memory sig0 = hex"e9be13c2658fcc1769b5e85e6e238a906fbf0d37c52a4ad5f4c69ff18156f3cb093a78ec7e848ab0649cfd1de13cfa4f2b215e42596f9c415541ec813b4626b401";
-        bytes memory sig0 = hex"cba766e2ba024aad86db556635cec9f104e76644b235f77759ff80bfefc990c5774d2d5ff3069a5099e4f9fadc9b08ab20472e2ef432fba94498d93c10cc584b00";
+        bytes memory sig = hex"cba766e2ba024aad86db556635cec9f104e76644b235f77759ff80bfefc990c5774d2d5ff3069a5099e4f9fadc9b08ab20472e2ef432fba94498d93c10cc584b00";
+        bytes memory prefix = "";
+        bytes memory message = hex"54686520717569636b2062726f776e20666f782064696420736f6d657468696e67";
```
Comment (vtleonardo) [inline src/facets/AccusationLibrary.t.sol:16]: Is this a hash of a Pclaims that had "Proposal" appended to it?

**Review by vtleonardo — APPROVED** [review]
Everything looks good to me!

**Review by ricardopinto — APPROVED** [review]
looks good to me

File: src/facets/AccusationLibrary.t.sol (hunk 1 above)
Comment (anthonygd) [reply src/facets/AccusationLibrary.t.sol:16]: No, this was just the string "The quick brown fox did something" with no prefix. The test `testSignedPClaims()` is a real pclaims with "Proposal" prepended.

**Pull request merged**